discovery-provider/src/api/v1/helpers.py

```diff
@@ -305,6 +305,16 @@ def extend_supporting(support: SupportResponse):
     }
 
 
+def extend_tip(tip):
```

Comment:
nit: let's type `tip`

Reply:
will probably do in my testing fast-follow, gonna leave for now since my working state is kinda messy